scripts/install_python_toolchain.sh

```diff
@@ -51,11 +54,11 @@ function download_file {
   fi
 } # end of download file
 
-$python_scripts/$PIP install --upgrade "pip>=8.1"
-$python_scripts/$PIP install -r scripts/requirements.txt
+$PIP install --upgrade "pip>=8.1"
+$PIP install -r scripts/requirements.txt
 if [[ $OSTYPE == darwin* ]]; then
   # macOS only test dependency
-  $python_scripts/$PIP install pyobjc==4.2.2
+  $PIP install pyobjc==4.2.2
```

Comment:
We could use platform markers in the `requirements.txt` file for this. This has been supported well before pip 8.1, so it should be perfectly safe to use. (Also, I'm a big fan!)

The line in `requirements.txt` would simply look like this:

```
pyobjc==4.2.2; sys_platform == 'darwin'
```

Reply:
Oh, good to know, thanks! I'll move that to a platform marker.